ray/helm/kuberay-operator/Chart.yaml

```diff
@@ -3,8 +3,8 @@ name: kuberay-operator
 description: A Helm chart for Kubernetes
 type: application
 version: 0.1.3
-appVersion: "v0.3.0"
+appVersion: "v0.6.0"
```

Comment:
appVersion here should be `v2.7.0` (the version of ray itself)

Reply:
in `ray/helm/ray/Chart.yaml` i have the appVersion set to `2.7.0`. since this chart.yaml is for kuberay i put the kuberay operator version, i can switch this to `2.7.0` but just want to double check

Reply:
Oh i didn't realize, that's probably fine.